google/cloud/storage/tests/testbench.py

```diff
@@ -611,6 +611,25 @@ def buckets_insert():
     return json.dumps(bucket.metadata)
 
 
+@gcs.route('/b/<bucket_name>', methods=['PUT'])
+def buckets_update(bucket_name):
+    """Implement the 'Buckets: update' API: update an existing Bucket."""
+    base_url = flask.url_for('gcs_index', _external=True)
+    insert_magic_bucket(base_url)
+    payload = json.loads(flask.request.data)
+    print("\n\n\n%s\n\n\n" % flask.request.data)
```

Comment:
I'm assuming this print is leftover from debugging?

Reply:
Indeed, thanks for catching that.  Removed.